libnavigation-sku/src/main/java/com/mapbox/navigation/sku/accounts/MapboxNavigationAccounts.kt

```diff
@@ -7,7 +7,7 @@ import com.mapbox.android.accounts.navigation.sku.v1.SkuGenerator
 import com.mapbox.android.accounts.navigation.sku.v1.TripsSku
 import com.mapbox.android.accounts.v1.AccountsConstants.MAPBOX_SHARED_PREFERENCES
 
-internal class MapboxNavigationAccounts private constructor() {
+class MapboxNavigationAccounts private constructor() {
```

Comment:
This class should be under an `internal` package as it's not meant to be used by external developers. I'm going to cut a ticket to audit the code and do the same across all modules.

Reply:
The only exposed methods are `startNavigation`, `stopNavigation` and `buildUrl`. They need to be public and we can't hide them if we want the SKU logic to live in a separate library.

Reply:
Yeah, and that's 🆗 What I meant is that they should live under an `internal` package (and be `public` of course) so developers know that this class is meant to be internal and they shouldn't rely on it. That also give us the possibility to change APIs and break these internal APIs "without" breaking SemVer. It's a common practice in SDKs / libraries development 👀 https://github.com/square/okhttp/tree/master/okhttp/src/main/java/okhttp3/internal

Reply:
Will cut a ticket describing ☝️ as I mentioned so we can follow up 🚀

Reply:
I missed `package` word :upside_down_face:

Reply:
Follow up ticket https://github.com/mapbox/mapbox-navigation-android/issues/2417